sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/CreateQueueOptions.cs

```diff
@@ -324,7 +324,7 @@ public override bool Equals(object obj)
         /// <summary>Determines whether the specified object is equal to the current object.</summary>
         public bool Equals(CreateQueueOptions other)
         {
-            if (other is CreateQueueOptions otherOptions
+            if (other is { } otherOptions
```

Comment:
Do we need this check/alias here.   Looks like the parameter is already typed to what we're looking for.

Reply:
As far as I understand this was intended to do a null check. If other is null the checks are bypassed and false is returned otherwise all properties are checked. Would you prefer `is not null`?

Reply:
Ah, that makes sense but was not obvious from the context.   Personally, I like the more explicit version from the changes as the intent is clear at a glance.

Reply:
Are you also OK for me to simplify it to a single return?